frontend/src/routes/_account.plan.index.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright 2025 New Vector Ltd.
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+// Please see LICENSE in the repository root for full details.
+
+import { queryOptions, useSuspenseQuery } from "@tanstack/react-query";
+import { Navigate, createFileRoute } from "@tanstack/react-router";
+import { useCallback, useEffect, useRef, useState } from "react";
+import { graphql, useFragment } from "../gql";
+import { graphqlRequest } from "../graphql";
+
+export const CONFIG_FRAGMENT = graphql(/* GraphQL */ `
+  fragment PlanManagement_siteConfig on SiteConfig {
+    planManagementIframeUri
+  }
+`);
+
+const QUERY = graphql(/* GraphQL */ `
+  query SiteConfig {
+    siteConfig {
+      ...PlanManagement_siteConfig
+    }
+  }
+`);
+
+export const query = queryOptions({
+  queryKey: ["siteConfig"],
+  queryFn: ({ signal }) => graphqlRequest({ query: QUERY, signal }),
+});
+
+export const Route = createFileRoute("/_account/plan/")({
+  loader: ({ context }) => context.queryClient.ensureQueryData(query),
```

Comment:
nit perf thing, but we could preload the iframe document here, e.g

```suggestion
  loader: async ({ context }) => {
    const { planManagementIframeUri } = await context.queryClient.ensureQueryData(query);
    if (planManagementIframeUri) 
      preload(planManagementIframeUri, { as: "document" });
  }
```

with `import { preload } from "react-dom";`